### PR TITLE
Feature 

### PR DESCRIPTION
**Pull request by AbdifatahYasin1**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            <li>&#8226 <b>1933</b> - Leaves his family's farm to attend the University of Minnesota, thanks<br> to a Depression era program known as the "National Youth Administration"<br></li>
+            <br>
+            <li>&#8226 <b>1935</b> - Has to stop school and save up more money. Works in the Civilian<br> Conservation Corps, helping starving Americans. "I saw how food changed<br> them", he said. "All of this left scars on me."<br></li>
+            <br>
+            <li> <b>&#8226 1914</b> - Born in Cresco, Iowa</li><br>
+            <li>&#8226 <b>1933</b> - Leaves his family's farm to attend the University of Minnesota, thanks<br> to a Depression era program known as the "National Youth Administration"<br></li>
+            <br>
+            <li>&#8226 <b>1935</b> - Has to stop school and save up more money. Works in the Civilian<br> Conservation Corps, helping starving Americans. "I saw how food changed<br> them", he said. "All of this left scars on me."<br></li>
+            <br>
+
```
Comment (AbdifatahYasin1) [inline index.html:47]: All of this gap is not allowed.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+<body>
+
+    <nav>
+        <h1>Dr. Borlaug
+        </h1>
+        <h1>The man who saved a billion lives</h1>
+    </nav>
+
+    <div class="background-image">
```
Comment (AbdifatahYasin1) [inline index.html:17]: use HTML5 semantic tags (e.g., header, nav) over divs.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            <li>&#8226 <b>1933</b> - Leaves his family's farm to attend the University of Minnesota, thanks<br> to a Depression era program known as the "National Youth Administration"<br></li>
+            <br>
+            <li>&#8226 <b>1935</b> - Has to stop school and save up more money. Works in the Civilian<br> Conservation Corps, helping starving Americans. "I saw how food changed<br> them", he said. "All of this left scars on me."<br></li>
+            <br>
+            
+            
+            
+            
+            
+        </ul>
```
Comment (AbdifatahYasin1) [inline index.html:52]: please fill the gap from line 47 to 51 .Do not add blank lines, spaces, or indentations without a reason.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,53 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
```
Comment (AbdifatahYasin1) [inline style.css:7]: Delete the body or put some styles in it  because it is not allowed empty body

**Review by AbdifatahYasin1 — COMMENTED** [review]
Action is required 🛑 

There is no any linter files you have to use linter 🛑

File: index.html (hunk 2 above)
Comment (tufoinnkuo10) [reply index.html:17]: - [ ] please always Remember not to be authoritative.. you are reviewing and suggesting changes not imposing them.

File: index.html (hunk 3 above)
Comment (tufoinnkuo10) [reply index.html:52]: - [ ] kindly give a reason (WHY) for the changes requested

File: style.css (hunk 4 above)
Comment (tufoinnkuo10) [reply style.css:7]: - [ ] please give a reason for this change u demanded. why will it be necessary to delete the body:?

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @AbdifatahYasin1 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ Great job pointing out the issues
✅ Good use of inline comments



Required Changes ♻️
Check the comments under the review
- [ ] Please always comment if there is an absence of the pull request message/summary. it helps in summarising what was done in the project
- [ ] kindly give your final review message if they should make changes or if you approve of the project

tufoin
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: style.css (hunk 4 above)
Comment (AbdifatahYasin1) [reply style.css:7]: Done

File: index.html (hunk 3 above)
Comment (AbdifatahYasin1) [reply index.html:52]: Done

File: index.html (hunk 2 above)
Comment (AbdifatahYasin1) [reply index.html:17]: Done

**Review by OmarRamoun — CHANGES_REQUESTED** [review]
# 🟡 Just One More Thing 🔸

## Hi @AbdifatahYasin1 👋

Great work on making the changes requested ♻️ by a previous reviewer 👏.

You've done well implementing some of the requested changes 💯.

Though, There are some **few** issues 🐛 that you still need to work on to go to the next project but you are almost there! 🏃

## Don't Feel frustrated 🤔

I am sure you worked hard 🛠️ to make these changes and actually you have to be proud of them 🥇.

And I understand how much you wanted to get this project approved ✔️ (I was at your place one day 😉). But, don't worry, You still can do it! 💪.

Just apply the changes ♻️ down below 👇 and this time it is going to be approved ✅. Just make sure that you **don't miss any of the changes** requested or introduce any **new bugs** 🐛.

> **Note:** A good tip is to check the box next to every change after you implement it ☑️. And you can also check the full website to see if it is still working after you have implemented the changes.

*Please, reach to [me](#thank-you-for-your-time-🙏-and-call-me-if-you-need-help-or-clarification-☎️) for any help 🪝 or any question ❓. I am always happy to help you 😃.*

## Required Changes ♻️

- [ ] [REQUIRED] Please, make sure to add a comment about how the student didn't add a PR description.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbdifatahYasin1 — COMMENTED** [review]
The request changed

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @AbdifatahYasin1 

Your project is complete! Great job for adding the requested changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- great review comments

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**